pages/index.js

```diff
@@ -1,17 +1,19 @@
 import Head from "next/head";
 import dynamic from "next/dynamic";
 import "../styles";
```

Comment:
don't think you still need this, no?

Reply:
Thank you man for telling me that. I don´t need import "../styles";